From 3ce74f778f173943811a78d820c891b67f404aa8 Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@nxp.com>
Date: Mon, 16 Jul 2018 09:18:59 -0500
Subject: [PATCH 4330/5242] MLK-18904-8: imx: hdp: Adding support for i.MX8M
 (MCU2) DP PHY to Makefile

commit  8f16685f38e236f88121ada52eba924243478184 from
https://source.codeaurora.org/external/imx/linux-imx.git

Adding API_AFE_mcu2_dp.o to the build.

Signed-off-by: Oliver Brown <oliver.brown@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/imx/hdp/Makefile b/drivers/gpu/drm/imx/hdp/Makefile
index 0f00b44..d0f8ab8 100644
--- a/drivers/gpu/drm/imx/hdp/Makefile
+++ b/drivers/gpu/drm/imx/hdp/Makefile
@@ -5,4 +5,5 @@ obj-$(CONFIG_DRM_IMX_HDP) += imx-hdp.o \
 			ss28fdsoi_hdmitx_table.o \
 			t28hpc_hdmitx_table.o \
 			API_AFE_mcu1_dp.o \
+			API_AFE_mcu2_dp.o \
 			imx-hdp-audio.o
-- 
1.7.9.5

